etc/helm/pachyderm/templates/pachd/deployment.yaml

```diff
@@ -471,20 +471,31 @@ spec:
             command:
             - /pachd
             - --readiness
+          initialDelaySeconds: {{ .Values.pachd.readinessProbe.initialDelaySeconds | default 0 }}
```

Comment:
I feel like we usually put these values in the default values.yaml file.  Helm's documentation says:

> In an actual chart, all static default values should live in the values.yaml, and should not be repeated using the default command (otherwise they would be redundant). However, the default command is perfect for computed values, which cannot be declared inside values.yaml.

These aren't computed, so should probably live in values.yaml and not in the code.